exercises/sum-of-multiples/canonical-data.json

```diff
@@ -128,6 +128,15 @@
       },
       "expected": 0
     },
+    {
+      "description": "the factor 0 does not affect the sum of multiples of other factors",
+      "property": "sum",
+      "input": {
+        "factors": [3,0],
```

Comment:
the comma! (a space after it)

Reply:
Please add the space of the comma @sshine.  

Your beta suggestion was there @petertseng when I last commented.  Good eye noticing it now missing.. :+1:

Reply:
That feature "went away" and took @petertseng's change with it.  :-1: